pkg/plugininventory/sqlite_inventory.go

```diff
@@ -56,22 +55,22 @@ type inventoryDBRow struct {
 	uri                string
 }
 
-// NewSQLiteBackend returns a new DiscoveryBackend using the data found at 'inventoryDir'.
-func NewSQLiteBackend(discoveryName, inventoryDir, prefix string) DiscoveryBackend {
-	return &SQLiteBackend{
+// NewSQLiteInventory returns a new PluginInventory connected to the data found at 'inventoryDir'.
+func NewSQLiteInventory(discoveryName, inventoryDir, prefix string) PluginInventory {
+	return &SQLiteInventory{
 		discoveryName: discoveryName,
```

Comment:
`discoveryName` should be independent of the `SQLiteInventory` operations. 

A user should not need to configure `discoveryName` when creating `SQLiteInventory` as this we might use for `InsertPlugin` and `UpdatePlugin` which doesn't require `discoveryName`.

Reply:
Good point!  Since this PR is closed, I'll make the change in #64 which is meant to improve this code already.